EIPS/draft_token_validation.md

```diff
@@ -0,0 +1,181 @@
+## Preamble
+
+    EIP: <to be assigned>
+    Title: Token Validation
+    Author: Tom Carchrae<tom@finhaven.com>, Gleb Naumenko<gleb@finhaven.com>, Brooklyn Zelenka<brooklyn@finhaven.com>
+    Type: Standard Track
+    Category: ERC
+    Status: Draft
+    Created: 2018-02-14
+
+## Simple Summary
+A protocol for services providing financial transaction validation.
+
+## Abstract
+This standard provides a registry contract method for authorizing token transfers.
+By nature, this covers both initially issuing tokens to users (ie: transfer from contract to owner),
+transferring tokens between users, and token spends.
+
+## Motivation
+The tokenization of assets has wide application,
+not least of which is financial instruments such as securities.
+Most jurisdictions have placed legal constraints on what may be traded,
+and who can hold such tokens which are regarded as securities. Broadly this includes KYC and AML validation,
+but may also include time-based spend limits, total volume of transactions, and so on.
+
+Regulators and sanctioned third-party compliance agencies need some way to link
+off-chain compliance information such as identity and residency to an on-chain service.
+The application of this design is broader than legal regulation, encompassing all manner
+of business logic permissions for the creation, management, and trading of tokens.
+
+## Specification
+
+### TokenValidator
+
+```solidity
+interface TokenValidator {
+  function check(
+    address _token,
+    address _user
```

Comment:
🤔 Is `_address` too general, though? I get that it may represent an autonomous contract in addition to human users. Perhaps `_subject` or `_toCheck`? (ie: the one being checked?)